imperatives/imperatives.go

```diff
@@ -111,7 +110,19 @@ var errFmtAddRewriter = errors.New("addRewriter <old> <new> <max>")
 var errFmtModDest = errors.New("modDest <routeKey> <dest> <addr/prefix/sub/regex=>") // one or more can be specified at once
 var errFmtModRoute = errors.New("modRoute <routeKey> <prefix/sub/regex=>")           // one or more can be specified at once
 
-func Apply(table *tbl.Table, cmd string) error {
+type Table interface {
```

Comment:
I need to import imperatives in table to be able to parse the destinations, so this avoids the circular import issue.  Since this file only ever uses a passed-in table object there's no reason to have an explicit dependency vs just accepting anything that implements the interface.